packages/jest-reporters/src/notify_impl.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+export default function notify(
+  notification: Notification,
+  callback?: NotificationCallback,
+) {
+  try {
+    return require('node-notifier').notify(notification, callback);
```

Comment:
I couldn't understand how the plugin made some imports lazy so I changed the code to call `require` when the `notify` function is called. 
 
`require` has it's own cache so I didn't think it was necessary to include `loadNotify()`

```javascript
function loadNotifier() {
  const data = require('node-notifier');
  loadNotifier = function() {
    return data;
  }
  return data; 
}
```

Reply:
that's fair 🙂 a `loadNotifier` would be way easier to type, though, as you could use `function loadNotifier(): typeof import('node-notifier')`